packages/roosterjs-editor-core/test/coreApi/getPendableFormatStateTest.ts

```diff
@@ -15,10 +15,31 @@ describe('getPendableFormatState', () => {
 
     it('underline with u tag', () => {
         const core = createEditorCore(div, {});
-        div.innerHTML =
-            '<div style="font-family: arial; font-size: 12pt; color: black; background-color: white"><u>test</u></div>';
-        core.api.focus(core);
-        const format = getPendableFormatState(core, false);
+
+        core.api.setContent(
```

Comment:
Consider write a common "runTest" function to extract all common logic to make the test code easier to read.